mungegithub/github/github.go

```
@@ -726,39 +747,42 @@ func (obj *MungeObject) RemoveLabel(label string) error {
 // GetHeadAndBase returns the head SHA and the base ref, so that you can get
 // the base's sha in a second step. Purpose: if head and base SHA are the same
 // across two merge attempts, we don't need to rerun tests.
-func (obj *MungeObject) GetHeadAndBase() (headSHA, baseRef string, ok bool) {
+//CHECK-UPDATED
+func (obj *MungeObject) GetHeadAndBase() (headSHA, baseRef string, ok bool, err error) {
 	pr, err := obj.GetPR()
 	if err != nil {
-		return "", "", false
+		glog.Errorf("Error: %v", err)
+		return "", "", false, err
 	}
 	if pr.Head == nil || pr.Head.SHA == nil {
-		return "", "", false
+		return "", "", false, err
 	}
 	headSHA = *pr.Head.SHA
 	if pr.Base == nil || pr.Base.Ref == nil {
-		return "", "", false
+		return "", "", false, err
 	}
 	baseRef = *pr.Base.Ref
-	return headSHA, baseRef, true
+	return headSHA, baseRef, true, err
 }
 
 // GetSHAFromRef returns the current SHA of the given ref (i.e., branch).
-func (obj *MungeObject) GetSHAFromRef(ref string) (sha string, ok bool) {
+//CHECK-UPDATED
+func (obj *MungeObject) GetSHAFromRef(ref string) (sha string, ok bool, err error) {
 	commit, response, err := obj.config.client.Repositories.GetCommit(obj.config.Org, obj.config.Project, ref)
 	obj.config.analytics.GetCommit.Call(obj.config, response)
 	if err != nil {
 		glog.Errorf("Failed to get commit for %v, %v, %v: %v", obj.config.Org, obj.config.Project, ref, err)
-		return "", false
+		return "", false, err
 	}
 	if commit.SHA == nil {
-		return "", false
+		return "", false, err
 	}
-	return *commit.SHA, true
+	return *commit.SHA, true, err
 }
 
 // SetMilestone will set the milestone to the value specified
 func (obj *MungeObject) SetMilestone(title string) error {
-	milestones := obj.config.ListMilestones("all")
+	milestones, _ := obj.config.ListMilestones("all")
```

Let's just return the error here